mlrun/run.py

```diff
@@ -367,7 +367,7 @@ def import_function(url="", secrets=None, db="", project=None, new_name=None):
         if not runtime:
             raise KeyError(f"function {name}:{tag} not found in the DB")
     else:
-        url, is_hub_uri = extend_hub_uri_if_needed(url)
+        url, is_hub_uri = extend_hub_uri_if_needed(url, db)
```

Comment:
seems like db is expected to be a string
```
:param db: optional, mlrun api/db path
```

Reply:
few lines up it changes it to rundb db is a string

Reply:
you are in the else clause

Reply:
oh, right right